samoa-api/src/main/java/org/apache/samoa/moa/streams/AvroFileStream.java

```diff
@@ -46,8 +46,8 @@
   private static final Logger logger = LoggerFactory.getLogger(AvroFileStream.class);
 
   public FileOption avroFileOption = new FileOption("avroFile", 'f', "Avro File(s) to load.", null, null, false);
-  public IntOption classIndexOption = new IntOption("classIndex", 'c',
-      "Class index of data. 0 for none or -1 for last attribute in file.", -1, -1, Integer.MAX_VALUE);
+  /*public IntOption classIndexOption = new IntOption("classIndex", 'c',
```

Comment:
Yes we do but since we also need in ArffFileStream, I moved up to the common FileStream parent.